src/http/context.zig

```diff
@@ -15,18 +16,21 @@ const Mime = @import("mime.zig").Mime;
 const _SSE = @import("sse.zig").SSE;
 
 const Runtime = @import("tardy").Runtime;
-const Task = @import("tardy").Task;
 const TaskFn = @import("tardy").TaskFn;
 
 const raw_respond = @import("server.zig").raw_respond;
 
 // Context is dependent on the server that gets created.
-pub fn Context(comptime Server: type) type {
+pub fn Context(comptime Server: type, comptime UserState: type) type {
```

Comment:
I think `AppState` is a more accurate name for this than `UserState` just because it highlights that the state is shared across the application.